tests/integration/multiple-stores-test.js

```diff
@@ -136,3 +136,23 @@ test("embedded records should be created in multiple stores", function(assert) {
     assert.equal(env.store_b.hasRecordForId("super-villain", "1"), true, "superVillain should exist in store:store-b");
   });
 });
+
+
+test("each store should have a unique instance of the serializers", function(assert) {
+  env.registry.register('serializer:home-planet', DS.RESTSerializer.extend({}));
+
+  let serializer_a = env.store_a.serializerFor('home-planet');
+  let serializer_b = env.store_b.serializerFor('home-planet');
+
+  assert.notEqual(serializer_a, serializer_b);
```

Comment:
Can you add more assertions here?

Specifically, we don't _actually_ care that they aren't the same instance (not really at least), what we care about is that `.store` on them is the right thing.

```js
assert.equal(get(serializer_a, 'store'), env.store_a);
assert.equal(get(serializer_b, 'store'), env.store_b);
```

Reply:
Updated